sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -691,6 +691,7 @@ NOT: 'NOT';
 EXISTS: 'EXISTS';
 BETWEEN: 'BETWEEN';
 LIKE: 'LIKE';
+ILIKE: 'ILIKE';
```

Comment:
New keyword that is not added to `nonReserved` is a breaking change. We either have to mention this in the release notes or add it to `nonReserved`